tools/distpartitioning/initialize.py

```diff
@@ -490,10 +67,12 @@ def multi_dev_init(params):
     params : argparser object
         argparser object providing access to command line arguments.
     """
+    rank = int(os.environ["RANK"])
+    
     #init the gloo process group here. 
-    dist.init_prcess_group("gloo", rank=params.rank, world_size=params.world_size)
-    print('[Rank: ', params.rank, '] Done with process group initialization...')
+    dist.init_process_group("gloo", rank=rank, world_size=params.world_size, timeout=timedelta(seconds=5*60))
+    print('[Rank: ', rank, '] Done with process group initialization...')
 
     #invoke the main function here.
-    proc_exec(params.rank, params.world_size, params)
-    print('[Rank: ', params.rank, '] Done with Distributed data processing pipeline processing.')
+    gen_dist_partitions(rank, params.world_size, params)
```

Comment:
what does `gen_dist_partitions` do? the naming is also very confusing.

Reply:
The idea is to generate distributed partitions, "gen_dist_partitions". This is the function which performs all the steps by our algorithm (data loading, data shuffling, and creating dgl objects). Lets target function renaming in the next iteration of code drop.